web/src/pages/Route/List.tsx

```diff
@@ -63,6 +104,84 @@ const Page: React.FC = () => {
     });
   };
 
+  const handleExport = (exportFileType: number) => {
```

Comment:
number is not clear enough IMO, could we use keyof or something else to have a clearer typing?

Reply:
used  `ExportFileType` instead, thanks a lot